src/delta/delta.c

```diff
@@ -399,11 +399,11 @@ static int should_skip_path(const char *prefix, const char *suffix) {
 }
 
 static int process_suffix(const char *suffix, const char *onlyprefix) {
+        const char *dropins;
         const char *p;
         char *f, *key;
         OrderedHashmap *top, *bottom, *drops, *h;
         int r = 0, k, n_found = 0;
-        bool dropins;
```

Comment:
oh, just leave this as bool, even if nulstr_contains() returns a const char* now, just convert it to a bool with `!!`, i.e.:

```c
…
bool dropins;
…
dropins = !!nulstr_contains(have_dropins, suffix)
…
```